src/zenml/utils/time_utils.py

```diff
@@ -136,3 +138,39 @@ def expires_in(
     if expires_at < now:
         return expired_str
     return seconds_to_human_readable(int((expires_at - now).total_seconds()))
+
+
+def validate_cron_expression(
```

Comment:
I remember that some of the orchestrators we use support a custom cron syntax (or at least supported). This is probably something that we have to take into account here?